src/docs/wtperf.dox

```diff
@@ -128,117 +128,6 @@ configuration options:
 DO NOT EDIT: THIS PART OF THE FILE IS GENERATED BY dist/s_docs.
 \endif
 
```

Comment:
Something unexpected happened here in your branch. This file should not change.

Reply:
Yes.  I've (re)committed the correct version.